docs/source/learners/retrieval_model1.rst

```diff
@@ -0,0 +1,233 @@
+
+The experiments aimed to assess the effectiveness of both open-source and commercial LLMs in term typing and taxonomy discovery, following three stages: data augmentation, model training, and inference. For Task A (Term Typing), training data included an optional context sentence (only for WordNet), a lexical term, and one or more conceptual term types. The test set contained only the lexical term and optional context. For Task B (Taxonomy Discovery), training data consisted of type pairs {Ta, Tb}, where Ta is the parent and Tb the child, while the test set included only types, requiring identification of “is-a” relations.
+
+1) Data Collection: Term and type descriptions were gathered from public sources like Wikipedia via its API, followed by cleaning and structuring. Commercial LLMs with web search capabilities—GPT-4o, Claude-3, and Copilot—were accessed through APIs using zero-shot prompts to gather additional contextual information. Ontology datasets were accessed directly via APIs or downloads to obtain relevant context.
+
+2) Training (Domain-Specific Continual Training): Context information for terms and types was integrated into the training data. For GeoNames, context was collected for all terms and types, while for other datasets, only type-level context was used.
+
++---------+-------------------------------------------------------------------------------------------------------------------------------------+
+| **Task**| **Description**                                                                                                                     |
++=========+=====================================================================================================================================+
+| Task A  | Models were trained on terms and corresponding types from the competition data. Each type was assigned a unique label, forming a    |
+|         | dataset of term–label pairs for fine-tuning.                                                                                        |
++---------+-------------------------------------------------------------------------------------------------------------------------------------+
+| Task B  | Hierarchical relationships were transformed into a binary classification format. Positive samples (parent–child) were labeled as 1, |
+|         | and negative samples (reversed pairs) as 0, creating the final dataset for training.                                                |
++---------+-------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
This table should not be look like this. For documentation it is good to use the table format as you have used in llms4ol.rst file!